### PR TITLE
Added copy arguments to manipulation.py

### PR DESCRIPTION
**Pull request by Fayad-Alman**
Updating manipulation functions in NumPy and PyTorch to return optional copies of arrays instead of views

### CODE REVIEW

File: ivy/functional/ivy/manipulation.py [hunk 1]
```diff
@@ -707,7 +711,7 @@ def squeeze(
         b: ivy.array([3., 4., 5.])
     }
     """
-    return current_backend(x).squeeze(x, axis, out=out)
+    return current_backend(x).squeeze(x, axis,copy=copy, out=out)
```
Comment (ZoeCD) [inline ivy/functional/ivy/manipulation.py:714]: Hi! There is a lint error here: `ivy/functional/ivy/manipulation.py:714:46: E231 missing whitespace after ','`
Could you quickly fix that please?

**Review by ZoeCD — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/manipulation.py (hunk 1 above)
Comment (Fayad-Alman) [reply ivy/functional/ivy/manipulation.py]: Done!

**Comment by Fayad-Alman** [comment]
@ZoeCD Mind taking another look for approval before merging this?

**Review by ZoeCD — APPROVED** [review]
LGTM!

**Pull request merged**